src/routers/conclusions.py

```
@@ -0,0 +1,154 @@
+            if filters == {}:
+                filters = None
+
+        stmt = crud.get_documents_with_filters(
+            workspace_name=workspace_id,
+            filters=filters,
+            reverse=reverse or False,
+        )
+
+        return await apaginate(db, stmt)
+    except ValueError as e:
+        logger.warning(f"Failed to list conclusions: {str(e)}")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)